second-hand-b/src/main/java/com/codesquad/secondhand/command/domain/member/Likes.java

```diff
@@ -0,0 +1,31 @@
+package com.codesquad.secondhand.command.domain.member;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.ElementCollection;
+import javax.persistence.Embeddable;
+import lombok.NoArgsConstructor;
+
+@Embeddable
+@NoArgsConstructor
+public class Likes implements Serializable {
+
+    private static final long serialVersionUID = 1905122041950251207L;
+
+    @ElementCollection
+    private List<Long> productsId = new ArrayList<>();
```

Comment:
```suggestion
    private List<Long> productIds = new ArrayList<>();
```
사소한 거긴 한데 `s` 가 뒤로가는게 조금 더 정확해 보입니다.